simpleConditional/minMaxTwoNumber.py

```diff
@@ -3,3 +3,10 @@
 # Ex:
 #   minTwoNumber(12, 3) # return (3, 12)
 #   minTwoNumber(-2, 3) # return (-2, 3)
+from typing import Tuple
+
+def minMaxTwoNumber(n1: int, n2: int) -> Tuple[int, int]:
```

Comment:
1/ A function need to have a return statement in the outer most code block. This code you have here will not work in most other language. In fact, Python need to reorder this code for you to make it work.

2/ if you are going this far, why not just you ternary operation.

Reply:
ok.